### PR TITLE
fix: Bad PrometheusRule annotations for server down

### PR DESCRIPTION
**Pull request by TheBeege**
The suggested `PrometheusRule` annotations for the server down alert were copied from the flow run crashed or failed alert. This rewrites the annotations to be more accurate. 

Let me know if you'd prefer a different wording.

### CODE REVIEW

File: charts/prometheus-prefect-exporter/values.yaml [hunk 1]
```diff
@@ -87,8 +87,8 @@ prometheusRule:
   #   labels:
   #     severity: critical
   #   annotations:
-  #     summary: Flow Run {{ $labels.flow_name }} {{ $labels.state_name }} (Flow {{ $labels.flow_name }})
-  #     description: "Flow Run failed or crashed\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"
+  #     summary: Prefect server is down
+  #     description: The total number of Prefect server pods in the Prefect namespace is zero
```
Comment (mitchnielsen) [inline charts/prometheus-prefect-exporter/values.yaml:91]:
```suggestion
  #     description: There are no Prefect server pods in the "prefect" namespace
```

Just a slight rephrase for clarity, and downcases + quotes the namespace name. Feel free to tweak this as needed 👍🏼

**Review by mitchnielsen — CHANGES_REQUESTED** [review]
Thanks for catching this @TheBeege, just one small suggestion!

**Comment by TheBeege** [comment]
> Thanks for catching this @TheBeege, just one small suggestion!

Thanks for the suggestion! Updated

**Review by mitchnielsen — APPROVED** [review]

**Comment by mitchnielsen** [comment]
Thanks again @TheBeege 🤝

**Pull request merged**